cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/rest/resource/config/RestResourceConfig.java

```diff
@@ -44,9 +45,9 @@ public RestResourceConfig() {
         scanner.addIncludeFilter(new AnnotationTypeFilter(Provider.class));
         register(RequestContextFilter.class);
         register(MultiPartFeature.class);
-        registerResources("com.clicktravel.cheddar.rest.exception.mapper",
-                "com.clicktravel.cheddar.server.http.filter", "com.clicktravel.cheddar.server.rest.resource.status",
-                "com.clicktravel.services");
+        registerResources("com.clicktravel.cheddar.rest.exception.mapper", "com.clicktravel.cheddar.server.http.filter",
+                "com.clicktravel.cheddar.server.rest.resource.status", "com.clicktravel.services");
+        property(ServerProperties.LOCATION_HEADER_RELATIVE_URI_RESOLUTION_DISABLED, true);
```

Comment:
So the intention was to stop Jersey to resolve relative URIs in the Location http header against the request URI?